rllib/env/single_agent_env_runner.py

```diff
@@ -737,6 +744,24 @@ def _convert_to_tensor(self, struct) -> TensorType:
         else:
             return tree.map_structure(tf.convert_to_tensor, struct)
 
+    def _convert_info_to_list(self, infos: Dict[str, Any]) -> List[Dict[str, Any]]:
+        # NOTE: This method has been copied from gymnasium.wrappers.vector_list_info.py!
+        # We needed to implement our own converter, b/c the gymnasium one crashes upon
+        # construction with:
+        # `AttributeError: 'AsyncVectorEnv' object has no attribute 'get_wrapper_attr'`
```

Comment:
Shall we ping here JKTerry?

Reply:
Yes, I'll create an issue with them.

Reply:
Awesome, I like working together with other teams that push the RL field on their side.

Reply:
This seems to have been fixed in their latest 1.0.0a1 release! :) Upgrading RLlib to that version in this PR here ...
Awesome. Thanks @pseudo-rnd-thoughts, @jkterry1 and team!

Reply:
No worries, let us know if you run into anymore strange errors. Feel free to ping me or dm on discord